kaggle/kaggle_secrets.R

```diff
@@ -13,8 +13,17 @@ get_user_secret <- function(label) {
     if (KAGGLE_USER_SECRETS_TOKEN == '') {
       stop("Expected KAGGLE_USER_SECRETS_TOKEN environment variable to be present.", call. = FALSE)
     }
-    request_body <- list(JWE = KAGGLE_USER_SECRETS_TOKEN, Label = label)
-    response <- POST(paste0(KAGGLE_BASE_URL, GET_USER_SECRET_BY_LABEL_ENDPONT), body = request_body, encode = "json")
+    request_body <- list(Label = label)
+    auth_header <- paste0("Bearer ", KAGGLE_USER_SECRETS_TOKEN)
+    headers <- add_headers(c("X-Kaggle-Authorization" = auth_header))
+    response <- POST(
+      paste0(KAGGLE_BASE_URL,GET_USER_SECRET_BY_LABEL_ENDPONT),
```

Comment:
idem

Reply:
nit: missing a space after `,` to be consistent

Reply:
Done